relay-server/src/envelope/content_type.rs

```diff
@@ -35,6 +35,8 @@ pub enum ContentType {
     SpanV2Container,
     /// `application/vnd.sentry.items.trace-metric+json`
     TraceMetricContainer,
+    /// `application/vnd.sentry.attachment.v2`
+    AttachmentV2Container,
```

Comment:
nit: AttachmentV2 is not a container in the sense of Span V2, i.e. it does not contain multiple attachments in one item.

```suggestion
    AttachmentV2,
```